src/compiler/scala/tools/nsc/transform/patmat/MatchAnalysis.scala

```diff
@@ -246,10 +246,6 @@ trait MatchApproximation extends TreeAndTypeAnalysis with ScalaLogic with MatchT
       override def toString = s"T${id}C($prop)"
     }
 
-    class TreeMakersToPropsIgnoreNullChecks(root: Symbol) extends TreeMakersToProps(root) {
```

Comment:
there's a stale comment in `TreeMakersToProps` saying "overridden in TreeMakersToPropsIgnoreNullChecks"